Atomic/scan.py

```diff
@@ -87,6 +87,10 @@ def set_scanner():
             raise ValueError(yaml_error)
 
         self.results_dir = os.path.join(self.results, self.scanner, self.cur_time)
+        last_scanned_file = os.path.join(self.results, "last_scanned.json")
+        with open(last_scanned_file, 'w') as f:
+            json.dump(self.cur_time, f, indent=4)
+
 
```

Comment:
The persistent data file has the scan time in it and is json.  This seams like the most straight forward approach.